python/llm/src/bigdl/llm/transformers/convert.py

```diff
@@ -107,6 +109,19 @@ def _replace_with_low_bit_linear(model, qtype, modules_to_not_convert=None,
                         model._modules[name].requires_grad_(False)
 
                         module.weight = None
+        elif type(module) == nn.Embedding and name not in modules_to_not_convert:
```

Comment:
I don't think we want to do it unconditionally; maybe adding some config to control behavior?